src/services/signature_pool.cpp

```diff
@@ -60,9 +60,7 @@ bool SignaturePool::verifySignature(signer_id_type receiver_id,
     string merger_id_str = to_string(partial_block.merger_id);
     bytes_builder.append(merger_id_str);
```

Comment:
merger_id는 64비트이지만, 메시지를 주고 받을 때는 base64로 바꾸어서 주고 받습니다. (나중에 256비트로 늘릴지 몰라요.) merger_id_str는 decimal string이니까
```suggestion
    bytes_builder.appendDec(merger_id_str);
```
가 맞을 텐데, 아마 partial_block.merger_id가 uint64_t 라면, 
```suggestion
    bytes_builder.append(partial_block.merger_id);
```
이런 식으로 해도 됩니다. ID가 핸들링되는 부분이 많은데, 각기 다른 방식으로 다루고 있어서, 한번 의논 해봐야 할 것 같네요.